src/main/java/de/retest/web/RecheckWebOptions.java

```diff
@@ -19,12 +19,16 @@
 
 	private final AutocheckingCheckNamingStrategy checkNamingStrategy;
 	private final ScreenshotProvider screenshotProvider;
+	private final long autocheckingDelayMillis;
```

Comment:
Just a suggestion: In order for a more flexible unit, do we want to use `Duration`, so one could use `Duration.ofSeconds( 1 )`? Then we would not have to carry the unit with the variable.

Reply:
I think the only sensible values for the delay are between 0 and 2000 millis. I guess with `Duration` it would be too easy to use too big values. 3000 sounds like a lot 3sec not. But in general a good idea.